javascript/javascript3/week2/js3w2/js3w2_homework/js3w2-hw2_PromiseResolvesAfterSettime.js

```diff
@@ -0,0 +1,36 @@
+
+
+// Question2: Create a function that has one parameter: resolveAfter. Calling this function will return a promise that resolves after the resolveAfter seconds has passed.
+
+function callAfterProvidedTime(resolveAfter) {
+  return new Promise((resolve, reject) => {
+
+    const resolveMsg = `Fullfilled: logged out after ${resolveAfter} sec. `;
+    const rejectMsg = `Rejected: logged out after  ${resolveAfter} sec. `;
+
+    setTimeout(() => {
+      resolve(resolveMsg);// order of these two function matter?? seems yes !
+      reject(rejectMsg);// if reject function is placed before resolve  , it is executed first 
```

Comment:
it's better you make some error here, like:
` if (isNaN(time)) {
      reject(rejectMsg);
        }`